src/main/resources/liquibase/042-updateDB.xml

```diff
@@ -21,4 +21,23 @@
     </changeSet>
     <!-- prefix for gen2 prostate pts in dev: 1771332079 on prod: -763171247  -->
     <!-- prod i.instance_id > 20000 are actual new added initial_biopsies for migrated pts  -->
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="carrier_per_study_key" author="pegah">
+        <addColumn tableName="ddp_instance">
+            <column name="carrier_tracking_url" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="DDP-5445-tracking_history_table" author="pegah">
+        <createTable tableName="kit_tracking_history">
+            <column name="dsm_kit_request_id" type="INT(11)" >
```

Comment:
Each table should have its own auto-generated primary key, rather than relying on the PK of another table.  Can you add the pk column and then change this to a foreign key on kit_request?